actionlib_tools/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(actionlib_tools)
+
+find_package(catkin REQUIRED)
+
+catkin_package()
+
+catkin_install_python(PROGRAMS scripts/axclient.py scripts/axserver.py scripts/dynamic_action.py scripts/library.py
```

Comment:
`axclient` and `axserver` without the extension, please. Other cleanup can come later but we might as well get this done.

Reply:
Actually— there's one other item here. If you're declining to move `library` and `dynamic _action` into a proper setuptools package, then the way they are imported needs to be changed for Python 3 friendliness.

```from dynamic_action import DynamicAction```

Becomes:

```from .dynamic_action import DynamicAction```

Reply:
I'm not sure about this. Since the `axclient` and `axserver` are scripts, not modules, relative imports don't work. Additionally, I don't have a py3 ROS environment to test in, but using `from __future__ import absolute_import` in py2 seems to be working as expected without any changes.

Admittedly, I am somewhat fuzzy on the differences between python 2/3 absolute/relative imports, so I could totally be missing something.

Reply:
File extensions removed in 06d7547

Reply:
Hmm, okay, well I'd like to move that stuff into a proper module anyway, and this whole repo needs attention for Py3 compat regardless, so it's fine to leave it.

Reply:
Agreed, definitely needs a follow-up PR. I'll get that going once we finalize this.